subxt/tests/integration/main.rs

```diff
@@ -15,19 +15,26 @@
 // along with subxt.  If not, see <http://www.gnu.org/licenses/>.
 
 mod codegen;
+#[cfg(feature = "integration-tests")]
```

Comment:
Could we just have a single

`#![cfg(feature = "integration-tests")]`

at the top of the file, to cover it all?

Reply:
Yep, I think we could remove those entirely from here when we move it to its own `integration-tests` crate